### PR TITLE
[SC-8874] update push actions options

### PR DESCRIPTION
**Pull request by kallisti-dev**

### CODE REVIEW

**Review by wr0ngway — APPROVED** [review]
Looks good

**Comment by kallisti-dev** [comment]
waiting until tests are fixed before I merge this one

**Comment by kallisti-dev** [comment]
Dropping this PR and creating new one at #369 since tests can't run on forked branches

**Pull request closed without merging**
